### ISSUE TITLE
Provide manylinux wheel

### ISSUE THREAD

**Issue by sk-** [hajimes/mmh3#17]
It would be great if `mmh3` provided prebuilt wheels for linux. See https://github.com/pypa/manylinux

**Comment by honnibal** [comment]
If you do decide to build wheels, you might find this utility useful: https://github.com/explosion/wheelwright

Thanks for your work on this package, @hajimes !

**Comment by mxr** [comment]
+1, it would help with issues such as #20 too. Another option for building wheels is cibuildwheel (see docs [here](https://github.com/joerick/cibuildwheel/blob/719352b1d200a27f62a5f9b24d7bcb6a7c994f57/README.md#L105)). Thanks for considering this @hajimes!

**Comment by william-silversmith** [comment]
Thanks for this useful library! At least for myself, I build these wheels manually on whatever hardware I have available. Here's an example docker file: https://github.com/seung-lab/connected-components-3d/blob/master/Dockerfile

I then manually copy the wheels out of the docker and upload them with twine.

```bash
docker build . --tag hajimes/mmh3
docker run -v $PWD/dist:/output hajimes/mmh3 /bin/bash
...
$ cp wheelhouse/* /output
$ exit
...
twine upload dist/....
```

**Comment by hajimes** [comment]
I apologize for being absent from maintaining mmh3 and not replying for so long.

After some tries, I finally decided to rely on [cibuildwheel](https://github.com/joerick/cibuildwheel) recommended by @mxr, since it is actively developed, well-documented, and gets better and better in these years.

At the same time I completely rewrote my CI system and migrated from TravisCI/AppVeyor to GitHub Actions, so mmh3 will fully receive benefits from cibuildwheel. TravisCI is still good in the sense that, with cibuildwheel, it allows the generation of wheels on as many platforms as GitHub Actions does, but recently its policy was changed so that it discourages the free CI for open-source projects.

I'm glad to continue to release new wheels in the future when new Python versions or platforms are introduced. Many thanks for great support and suggestions!
